osf/models/user.py

```diff
@@ -1134,6 +1135,12 @@ def update_search_nodes(self):
         for node in self.contributed:
             node.update_search()
 
+    def update_date_last_login(self, save=False):
+        if not self.date_last_login or throttle_period_expired(self.date_last_login, website_settings.DATE_LAST_LOGIN_THROTTLE):
```

Comment:
The throttle can be removed from the property wrapper - this is only called during explicit one time authentication events and it seems safe to perform the update action.